server/src/main/java/org/opensearch/action/admin/cluster/node/info/NodeInfo.java

```diff
@@ -102,6 +103,9 @@ public NodeInfo(StreamInput in) throws IOException {
         if (in.getVersion().onOrAfter(LegacyESVersion.V_7_10_0)) {
             addInfoIfNonNull(AggregationInfo.class, in.readOptionalWriteable(AggregationInfo::new));
         }
+        if (in.getVersion().onOrAfter(Version.V_2_7_0)) {
```

Comment:
@msfroh we would need to bring these changes to `main` as well, could you please prepare the pull request so we will merge it right after this one? thank you

Reply:
Done, thanks! Here we go: https://github.com/opensearch-project/OpenSearch/pull/7135